src/battle-dex.ts

```diff
@@ -655,10 +655,20 @@ const Dex = new class implements ModdedDex {
 
 		if (options.shiny && mechanicsGen > 1) dir += '-shiny';
 
-		// April Fool's 2014
+		// April Fool's
```

Comment:
```suggestion
		// April Fool's 2014
```

please don't delete history